README.md

```diff
@@ -50,23 +50,25 @@ helm delete kubernetes-external-secrets
-| `image.tag`                          | kubernetes-external-secrets Image tag                        | `1.2.0`                                                 |
-| `image.pullPolicy`                   | Image pull policy                                            | `IfNotPresent`                                          |
-| `rbac.create`                        | Create & use RBAC resources                                  | `true`                                                  |
-| `serviceAccount.create`              | Whether a new service account name should be created.        | `true`                                                  |
-| `serviceAccount.name`                | Service account to be used.                                  | automatically generated
-| `podAnnotations`                     | Annotations to be added to pods                              | `{}`                                                    |
-| `replicaCount`                       | Number of replicas                                           | `1`                                                     |
-| `nodeSelector`                       | node labels for pod assignment                               | `{}`                                                    |
-| `tolerations`                        | List of node taints to tolerate (requires Kubernetes >= 1.6) | `[]`                                                    |
-| `affinity`                           | Affinity for pod assignment                                  | `{}`                                                    |
-| `resources`                          | Pod resource requests & limits                               | `{}`                                                    |
+| Parameter                                 | Description                                                  | Default                                                 |
+| ----------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------- |
+| `env.AWS_REGION`                          | Set AWS_REGION in Deployment Pod                             | `us-west-2`                                             |
+| `env.EVENTS_INTERVAL_MILLISECONDS`        | Set EVENTS_INTERVAL_MILLISECONDS in Deployment Pod           | `60000`                                                 |
+| `env.POLLER_INTERVAL_MILLISECONDS`        | Set POLLER_INTERVAL_MILLISECONDS in Deployment Pod           | `10000`                                                 |
+| `envVarsFromSecret.AWS_ACCESS_KEY_ID`     | Set AWS_ACCESS_KEY_ID (from a secret) in Deployment Pod      |                                                         |
+| `envVarsFromSecret.AWS_SECRET_ACCESS_KEY` | Set AWS_SECRET_ACCESS_KEY (from a secret) in Deployment Pod  |                                                         |
```

Comment:
Curious why `envVarsFromSecret` instead of `env`? The values can be provided via the helm chart, but they can also be provided by other ways (e.g. `export`). Is there a way to express both cases here?

Reply:
envVarsFromSecret is prob not a great name. Were you thinking maybe nesting it under env? I'm okay with that. I'm new to helm and I keep reading about how you should keep your configs as "flat" as possible- but new to helm and don't have much reference to draw from. Or did you just mean make the documentation clearer?